ivy/functional/backends/paddle/experimental/creation.py

```diff
@@ -223,3 +223,32 @@ def mel_weight_matrix(
         upper_edge_hertz,
     )
     return paddle.transpose(mel_mat, (1, 0))
+
+
+def polysub(poly1_coeffs: paddle.Tensor, poly2_coeffs: paddle.Tensor) -> paddle.Tensor:
+    """
+    Calculate the difference of two polynomials.
```

Comment:
There is no need for docstrings in our backend implementations. Can you please remove this?